classes/Product.php

```diff
@@ -2396,6 +2396,10 @@ public function getAttributesResume($id_lang, $attribute_value_separator = ' - '
             } else {
                 $combinations[$key]['quantity'] = Cache::retrieve($cache_key);
             }
+
+            $ecotax = (float) $combinations[$key]['ecotax'] ?? 0;
+            $combinations[$key]['ecotax_tax_excluded'] = $ecotax;
+            $combinations[$key]['ecotax_tax_included'] = Tools::ps_round($ecotax * (1 + Tax::getProductEcotaxRate() / 100), 6);
```

Comment:
Shouldn't the `6` be a const or took from a configuration?

Reply:
Yes, you're right!!